### PR TITLE
Tools: Topology2: Add topologies builds for PTL 

### PR DESCRIPTION
**Pull request by singalsu**

### CODE REVIEW

File: tools/topology/topology2/cavs-nocodec.conf [hunk 1]
```diff
@@ -142,6 +142,9 @@ Object.Dai.SSP [
 			Object.Base.link_config.1 {
```
Comment (singalsu) [inline tools/topology/topology2/cavs-nocodec.conf:142]: Note: this commit will be deleted, it's not used in the reference blobs.

**Comment by singalsu** [comment]
Update, I added build of DMIC 4 channels 96 kHz topologies.

File: tools/topology/topology2/platform/intel/dmic-generic.conf [hunk 2]
```diff
@@ -518,8 +518,17 @@ Object.PCM.pcm [
 			formats 'S32_LE'
 			channels_min $NUM_DMICS
 			channels_max $NUM_DMICS
-			rate_min $DMIC0_RATE
-			rate_max $DMIC0_RATE
+			IncludeByKey.DMIC0_RATE {
+				"16000" {
+					rates '48000'
```
Comment (ranj063) [inline tools/topology/topology2/platform/intel/dmic-generic.conf:523]: same here?

File: tools/topology/topology2/development/tplg-targets.cmake
Comment (RDharageswari) [inline tools/topology/topology2/development/tplg-targets.cmake:1]: Hi Seppo,
Just curious...why is the ssp2->headset required for PTL

File: tools/topology/topology2/cavs-nocodec.conf [hunk 3]
```diff
@@ -976,6 +1011,17 @@ IncludeByKey.PASSTHROUGH {
 					formats 'S32_LE'
 					channels_min $NUM_DMICS
 					channels_max $NUM_DMICS
+					IncludeByKey.DMIC0_RATE {
+						"16000" {
+							rates '48000'
```
[the remark this answers is not in the document]
Comment (singalsu) [reply tools/topology/topology2/cavs-nocodec.conf:1016]: Oops, yes need to fix!

File: tools/topology/topology2/development/tplg-targets.cmake
Comment (singalsu) [reply tools/topology/topology2/development/tplg-targets.cmake:1]: I only copied this from topologies of previous platforms? Is there no ssp2 in in PTL or not needed in products?

Comment (plbossart) [reply tools/topology/topology2/development/tplg-targets.cmake:1]: there are 3 SSPs as usual. The only open is which SSP is used for what, that depends on the board layout and the assignment of one SSP to BT offload.

**Comment by lgirdwood** [comment]
@plbossart @ranj063 @RDharageswari good for you now ?

**Comment by singalsu** [comment]
Fixed the conflict in dmic-generic.conf, no other changes.

**Review by lgirdwood — APPROVED** [review]
I think open is resolved ?

**Comment by lgirdwood** [comment]
@kv2019i pls review/merge

**Pull request merged**